tests/chainer_tests/functions_tests/pooling_tests/test_spatial_pyramid_pooling_2d.py

```diff
@@ -19,8 +19,14 @@ class TestSpatialPyramidPooling2D(unittest.TestCase):
 
     def setUp(self):
         # Avoid unstability of numerical gradient
-        self.x = numpy.random.randn(
-            self.n, self.c, self.h, self.w).astype(numpy.float32)
+        shape = (self.n, self.c, self.h, self.w)
+        size = numpy.prod(shape)
+        self.x = numpy.arange(size, dtype=numpy.float32).reshape(shape)
+        numpy.random.shuffle(self.x)
+        self.x += numpy.random.uniform(
+            0.4, 0.6, shape).astype(numpy.float32)
+        self.x /= size
```

Comment:
My understanding is that if `x > y` and `x + ε < y`, the result of max(x, y) is unstable. So we separate each element of `x` enough so that small ε does not change the order of elements.